ui/store/actions.js

```diff
@@ -406,6 +407,8 @@ export function connectHardware(deviceName, page, hdPath, t) {
   return async (dispatch, getState) => {
     const { ledgerTransportType } = getState().metamask;
 
+    const isPopup = getEnvironmentType() === ENVIRONMENT_TYPE_POPUP;
+
     dispatch(
       showLoadingIndication(`Looking for your ${capitalize(deviceName)}...`),
```

Comment:
`Looking for your ${capitalize(deviceName)}...` needs to be localized

Reply:
This isn't new -- it's an existing issue.  We don't have `this.context.t` available in actions.  I believe there's an issue for this open already.